### PR TITLE
Refactored Cilium Operations

### LINKED ISSUE #30: [Chore] Refactor Cilium Operations
#### Current Behavior

- We have two operations.go file inside of cilium repo. 
- [meshery-cilium/cilium/operations.go](https://github.com/meshery/meshery-cilium/blob/master/cilium/operations.go) is redundant 

#### Expected Behavior

- Its functionalities should be moved to [meshery-cilium/cilium/cilium.go](https://github.com/meshery/meshery-cilium/blob/master/cilium/cilium.go). So that we maintain consistency across all meshery adapters.
- Correspondingly, other references to it must be updated.

For reference:
Check other meshery adapters, like: [meshery-istio](https://github.com/meshery/meshery-istio), [meshery-linkerd](https://github.com/meshery/meshery-linkerd) etc.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by MUzairS15**
Signed-off-by: MUzairS15 <muzair.shaikh810@gmail.com>

**Description**

This PR fixes #30 

**Notes for Reviewers**
@lee added cilium operations as you mentioned earlier today.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
// @leecalcote @theBeginner86

**Comment by leecalcote** [comment]
Potentially, the final PR before the adapter moves to Beta...

File: cilium/cilium.go [hunk 1]
```diff
@@ -40,6 +44,83 @@ func New(config meshkitCfg.Handler, log logger.Handler, kc meshkitCfg.Handler) a
 	}
 }
 
+// ApplyOperation function contains the operation handlers
+func (h *Handler) ApplyOperation(ctx context.Context, request adapter.OperationRequest) error {
```
Comment (MUzairS15) [inline cilium/cilium.go:48]: Should I change structure name to be more specific `Cilium`??

Comment (leecalcote) [reply cilium/cilium.go:48]: @MUzairS15, ideally, not. Anything that can be generic, should be generic so as to promote consistency across adapters. // @Revolyssup

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**